api/src/main/java/jakarta/servlet/jsp/tagext/VariableInfo.java

```diff
@@ -72,12 +76,12 @@
  * The JSP 2.0 specification defines the interpretation of 3 values:
  * 
  * <ul>
- * <li>NESTED, if the scripting variable is available between the start tag and the end tag of the action that defines
- * it.
- * <li>AT_BEGIN, if the scripting variable is available from the start tag of the action that defines it until the end
- * of the scope.
- * <li>AT_END, if the scripting variable is available after the end tag of the action that defines it until the end of
- * the scope.
+ * <li>NESTED, if the scripting variable is synchronized with the calling page between the start tag and the end tag of
+ * the action that defines it.
+ * <li>AT_BEGIN, if the scripting variable is synchronized with the calling page from the start tag of the action that
+ * defines it until the end of the scope.
+ * <li>AT_END, if the scripting variable is synchronized with the calling page after the end tag of the action that
```

Comment:
The word "synchronized" in this context seems odd.
Can you explain what you mean here?

Reply:
That wording based on section 8.9.1 of the spec document.
What I was trying to do was make clear that "scope" in this case does not mean "scope in the sense of where is this variable declaration visible", but "how and when the scripting variable is kept in sync with the calling page".

Reply:
Is that explanation sufficient or are you looking for some changes in wording?